README.md

```diff
@@ -0,0 +1,94 @@
+$ cd gr-stellarstation
+$ mkdir build
+$ cmake ..
+$ make
+$ sudo make install
+$ sudo ldconfig
+```
+
+## Examples
+
+See the [examples directory](examples/) for flowgraphs showing the usage of gr-stellarstation blocks.
+
+## Stellarstation API Key
+
+To use most of the blocks in this module, you will need a Stellarstation API Key, which is a .json file containing credentials to access the Stellarstation API. You can request one from <https://www.stellarstation.com/>.
+
+## Quickstart
+
+A typical use case of gr-stellarstation is direct streaming of complex I/Q data from a satellite tracked by Stellarstation to a GNURadio flowgraph in real-time.
+
+gr-stellarstation provides the block **Stellarstation IQ Source** for this purpose.
+
+![iq_source_sample](docs/images/iq_source_sample.png)
+
+Note that we've configured the block to connect to *Satellite ID* "73" (corresponding to the weather satellite [NOAA 19](https://en.wikipedia.org/wiki/NOAA-19)) and given it the path of the Stellarstation API Key via *Stellarstation API Key Filepath*.
+
+When this flowgraph runs, the block opens a stream to the Stellarstation API, and converts all IQ packets received to a complex stream that can be used in downstream GNURadio blocks. We can display a live waterfall plot, save the stream to file, demodulate the data, or anything else GNURadio lets us do!
+
+Here are a few more things to keep in mind about our flowgraph:
+* The sampling rate of the received stream is satellite-dependent. Currently, most amateur satellites that Stellarstation tracks are sampled at 112500Hz.
+* If a pass is not currently running, the Stellarstation API will return no data, and your flowgraph will not do anything.
+* Depending on both the ground station's and the user's internet speed, samples may not be received in realtime. For example, to support streaming 112500Hz raw I/Q data in realtime, the ground station and the user need to support approximately ~1MBps upload and download speed, respectively. If either one is slower, the packets will still arrive in order without data loss, but  will no longer be in realtime.
+
+For other satellites, Stellarstation does demodulation directly on the ground station and streams back the demodulated bitstream. For users that wish to access this data, gr-stellarstation provides the **Stellarstation Bitstream Source** block.
+
+![bit_source_sample](docs/images/bit_source_sample.png)
+
+## Advanced Usage
+
+The **Stellarstation IQ Source** and **Stellarstation Bitstream Source** blocks are hierarchical blocks based on the core **Stellarstation API Source** block. It is responsible for connecting to the Stellarstation API via [this API call](https://github.com/infostellarinc/stellarstation-api/blob/master/api/src/main/proto/stellarstation/api/v1/stellarstation.proto#L69) and sending received packets as GNURadio PMTs in [PDU](https://wiki.gnuradio.org/index.php/Guided_Tutorial_Programming_Topics#5.3.1_PDUs) format for downstream blocks to consume.
```

Comment:
it's safer to not include exact line number in the link until API 1.0.0 since we will have more changes in the API definition.